src/data/icons.json

```diff
@@ -240,6 +250,7 @@
   { "name": "ChromeBackMirrored", "unicode": "EA47" },
   { "name": "ChromeClose", "unicode": "E8BB" },
   { "name": "ChromeMinimize", "unicode": "E921" },
+  { "name": "ChronosLogo", "unicode": "F541" },
```

Comment:
I'm waiting to hear back on whether or not this icon is intended for external use.

Reply:
Good catch!